### PR TITLE
Share portfolio modal - add description containing the portfolio name 

### PR DESCRIPTION
**Pull request by lgalis**
Share portfolio modal - add description containing the portfolio name 

Fixes https://projects.engineering.redhat.com/browse/SSP-1373

Screenshot from 2020-04-16 16-52-31

### CODE REVIEW

File: src/smart-components/portfolio/share-portfolio-modal.js [hunk 1]
```diff
@@ -17,7 +24,11 @@ import useQuery from '../../utilities/use-query';
 import useEnhancedHistory from '../../utilities/use-enhanced-history';
 import { UnauthorizedRedirect } from '../error-pages/error-redirects';
 
-const SharePortfolioModal = ({ closeUrl, removeQuery }) => {
+const SharePortfolioModal = ({
+  closeUrl,
+  removeQuery,
+  portfolioName = () => ''
```
Comment (Hyperkid123) [inline src/smart-components/portfolio/share-portfolio-modal.js:30]: Can we just pull in the portfolio name from the store with selector?

Comment (lgalis) [reply src/smart-components/portfolio/share-portfolio-modal.js:30]: that will work from the portfolio details page, but not from the kebab menu on the portfolio card

Comment (lgalis) [reply src/smart-components/portfolio/share-portfolio-modal.js:30]: @Hyperkid123 - that will work from the portfolio details page, but not from the kebab menu on the portfolio card

Comment (Hyperkid123) [reply src/smart-components/portfolio/share-portfolio-modal.js:30]: oh yeah there is multiple entry points

**Pull request merged**